Mathlib/Tactic/Simproc/Divisors.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2025 Paul Lezeau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Lezeau, Bhavik Mehta
+-/
+import Mathlib.NumberTheory.Divisors
+import Mathlib.Util.Qq
+import Mathlib.Data.Finset.Sort
+
+/-! # Divisor Simprocs
+
+This file implements simprocs to compute various objects related to divisors:
+- `Nat.divisorsEq`: computes `Nat.divisors n` for explicit values of `n`
+- `Nat.properDivisorsEq`: computes `Nat.properDivisors n` for explicit values of `n`
+
+-/
+
+--TODO: these simprocs can probably be made a lot more efficient. See the discussion in #23026.
+
+open Lean Meta Qq
+
+/-- The `Nat.divisorsEq` computes the finset `Nat.divisors n` when `n` is a natural number
+literal. -/
+simproc_decl Nat.divisorsEq (Nat.divisors _) := fun e => do
```

Comment:
I would argue this should be called `Nat.divisors_ofNat` since it is specifically matching `ofNat`.

Reply:
Done